database/dbase.go

```diff
@@ -139,6 +142,9 @@ func (cfg *DbConfig) OpenDatabase() (CloudtrustDB, error) {
 			dbConn.Close()
 			dbConn = nil
 		}
+	} else if cfg.ConnectionCheck {
+		// Executes a simple query to check that the connection is valid
+		_, err = dbConn.Exec("select 1")
```

Comment:
Not an expert here, just expressing my reasoning:
Is "select 1" a standard accepted sql query?
Couldn't we use dbConn.Ping() ?

Reply:
I was not aware such a method could exist... Thanks for this comment!
(usually, there is no ping method on database connections and we just execute a very simple request like `select 1` or `select 1 from dual`)